docs/source/troubleshooting.rst

```diff
@@ -226,9 +226,32 @@ On Ubuntu / Debian:
 .. code-block:: bash
 
     /usr/lib/python2.7/dist-packages/st2common/bin/st2-self-check
-    
+
 On RedHat / Fedora:
 
 .. code-block:: bash
 
     /usr/lib/python2.7/site-packages/st2common/bin/st2-self-check
+
+
+Support
+-------
+
+Support is available to you via `email <support@stackstorm.com/>`_ or via
+`slack-community <https://stackstorm.typeform.com/to/K76GRP)/>`_.
+
```

Comment:
Wait, says who?

I definitely think we should still use and support IRC. I'm not a fan of only supporting proprietary and closed source solutions like Slack.

In any case, I do need to say that I'm not a fan of now having that many support channels (it's hard to keep track of all of them). Imo, the solution is to create bi-direction relay / bridge between our main Slack instance, stackstorm-community and IRC.

I wanted to do this before, but it's currently not possible since we can only run one instance of each sensor (once @manasdk pull request is merged this should be possible).